### ISSUE TITLE
[Due for payment 2026-04-14] [$250][Accessibility][Tracking] Hide from Assistive Tech / Remove Interactive Role

### ISSUE THREAD

**Issue by rushatgabhane** [Expensify/App#79243]
## Overview

This issue tracks elements that should be hidden from assistive technology or have incorrect interactive roles removed.

## Context

Part of the accessibility improvement effort tracked in https://github.com/Expensify/Expensify/issues/575078

## Issues to Fix

- https://github.com/Expensify/App/issues/77497 - Toggle/checkbox label announced twice *(hide text with accessible={false})*
- https://github.com/Expensify/App/issues/77496 - Plain text elements announced as actionable *(remove interactive role)*

## Instructions

**Suggested Fix:** Hide redundant elements or remove incorrect roles

> **Note:** Discuss on Slack if you are unsure whether a sub-issue should be fixed or if you are unsure about the right fix / expected result. Ideally, create a single PR that will fix all the issues listed to minimize review time (more PRs only if necessary).
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009703243399104750</li>
        <li>Upwork Job ID: 2009703243399104750</li>
        <li>Last Price Increase: 2026-01-09</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by yatin-italy** [comment]
Thanks a lot! I’m happy to take on these issues. 😊

**Comment by rushatgabhane** [comment]
@dominictb are we close to merging the PR?

**Comment by Uzaifm127** [comment]
@dominictb @Valforte  Closed the previous PR due to some dangerous changes by mistake. Will open a new PR.

Extremely sorry for the mistake.

**Comment by rushatgabhane** [comment]
@dominictb can you take care of this please - https://github.com/Expensify/App/issues/79243#issuecomment-4106067219

**Comment by truph01** [comment]
@dominictb The [PR](https://github.com/Expensify/App/pull/86731) to fix Android mWeb is ready!
I will be the implementor for it (via Melvin bot).

**Comment by mallenexpensify** [comment]
Please comment and tag me if/when payment is due, thx

**Comment by mkhutornyi** [comment]
Please assign me as reviewer

**Comment by Uzaifm127** [comment]
@mallenexpensify Let me try to clarify the full journey of this issue because there were multiple PRs and regressions involved:

In the first [PR](https://github.com/Expensify/App/pull/80765), I worked on it and @dominictb reviewed it. That [PR](https://github.com/Expensify/App/pull/80765) solved one of the two issues mentioned by the OP. It also fixed a regression that had been introduced by another PR (not by our PR). We also covered the second issue in that [PR](https://github.com/Expensify/App/pull/80765) but we left the Android mweb case and then another [PR](https://github.com/Expensify/App/pull/81436) already fixed that before ours get merged.

Then for the second issue, I was working but I had to OOO so @truph01 took over.

Then they raised a [PR](https://github.com/Expensify/App/pull/86731) with Melvin and it merged then it got reverted [here](https://github.com/Expensify/App/pull/87062) based on the reasoning they provided [here](https://github.com/Expensify/App/issues/79243#issuecomment-4191832953).

After the revert, @truph01 raised an another final [PR](https://github.com/Expensify/App/pull/87291) which then solved the Android mweb issue we left.

Note: The first PR by @truph01 (before regression) was reviewed by @dominictb and second PR by @truph01 (after regression) was reviewed by @mkhutornyi 

---

I hope I explained well.

**Comment by Uzaifm127** [comment]
> ROLE: @Uzaifm127 paid $(AMOUNT) via Upwork (LINK)

@mallenexpensify Melvin is saying paid but I'm also due payment here.

**Comment by mallenexpensify** [comment]
### Payment Summary
Contributor: @Uzaifm127 paid $250 via Upwork
Contributor+: @truph01  due $125 via NewDot
Contributor+: @mkhutornyi  due $250 via NewDot
Contributor+: @dominictb due $250 via NewDot


@Uzaifm127 can you please accept the job below? Please reply here and tag me once you have. 
If you're not sent an offer in Upwork when you're assigned to an issue, and an Upwork job exists, please submit a proposal.
https://www.upwork.com/jobs/~022059029571903417582


> I hope I explained well.

@Uzaifm127, yes, the breakdown was very helpful (even if I might not have gotten the first attempt at the payment summary correct) 

All, comment if anything isn't correct. Provide reasoning and links, when needed and tag me too, thx

**Comment by Uzaifm127** [comment]
@mallenexpensify Accepted the offer, requested the payment. Thanks.

> please submit a proposal

Also, luckily I applied on the job and submitted the proposal when I got the offer.

**Comment by dominictb** [comment]
@mallenexpensify Hi this is my C+ work and outside of backend hourly work so I'm also due payment here.

**Comment by Uzaifm127** [comment]
Yes, @dominictb is correct. They started with this issue before they were the backend contributor and then in between they became the backend contributor but still they continued to review the PR.

**Comment by mallenexpensify** [comment]
@Uzaifm127 paid, summary updated above. Thx.
Added @dominictb too for the review of the first PR, think/hope we're good to close

**Comment by trjExpensify** [comment]
approved $250 for @dominictb

**Comment by trjExpensify** [comment]
approved $250 for @mkhutornyi

**Comment by garrettmknight** [comment]
$125 approved for @truph01
